src/GlobalLoop.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace React\EventLoop;
+
+final class GlobalLoop
+{
+    /**
+     * @internal
+     *
+     * @var LoopInterface
+     */
+    public static $loop;
```

Comment:
Makes sense, lets see what @clue, @cboden, and @nrk have to say and get the ball rolling if they like it :smile: